### DISCUSSION TITLE
form demo input control output error-> Uncaught Error: Permission denied to access property "toJSON"

### DISCUSSION THREAD

**Discussion by hunterdq** [ant-design-blazor/ant-design-blazor#1101] [Q&A]
https://antblazor.com/zh-CN/components/form
Uncaught Error: Permission denied to access property "toJSON"

**Comment by ElderJames** [comment]
Hi @hunterdq , which browser are you using?

**Reply by hunterdq** [reply]
Firefox  85.0 (64 )

**Reply by anddrzejb** [reply]
@hunterdq can you try with newest nightly build? I think this problem has been fixed by PR #1121.
